charts/linkerd2-multicluster/templates/gateway.yaml

```diff
@@ -11,12 +11,6 @@ data:
   nginx.conf: |-
     events {
     }
-    stream {
```

Comment:
How do we configure/expose this port? It seems like a lot of overhead to wire through the config for now. I thin the fallback case, we'd have to patch resources, replacing the nginx config.

Reply:
My understanding was that simply switching the value of the `gatewayPort` value in values.yaml would be enough to revert to the old behavior (since the service mirror controller discovers the gateway port by name)